eahub/templates/eahub/talentsearch.html

```diff
@@ -41,10 +41,10 @@
           {% thumbnail profile.image "200x200" crop="center" as im %}
```

Comment:
We should remove this page altogether! We forgot to remove it when we introduced the new profiles search:
- https://eahub.org/volunteers/  
- https://eahub.org/speakers/  
- https://eahub.org/candidates   

Can be done in a separate MR (but let's create a ticket to not forget)

Reply:
always happy to drop dead code - done 👍